.ci/bump-opbeans-ruby.yml

```diff
@@ -0,0 +1,80 @@
+---
+## Workflow to periodically check if there is an available newer APM agent version, e.g.
+## "1.2.3". If so, then update to it.
+name: Bump elastic-apm to latest version
+
+title: Bump elastic-apm to latest version
+
+scms:
+  elastic-apm-agent-ruby:
+    kind: github
+    spec:
+      user: '{{ requiredEnv "GIT_USER" }}'
+      email: '{{ requiredEnv "GIT_EMAIL" }}'
+      owner: elastic
+      repository: apm-agent-ruby
+      token: '{{ requiredEnv "GITHUB_TOKEN" }}'
+      username: '{{ requiredEnv "GIT_USER" }}'
+      branch: main
+  githubConfig:
+    kind: github
+    spec:
+      user: '{{ requiredEnv "GIT_USER" }}'
+      email: '{{ requiredEnv "GIT_EMAIL" }}'
+      owner: elastic
+      repository: opbeans-ruby
+      token: '{{ requiredEnv "GITHUB_TOKEN" }}'
+      username: '{{ requiredEnv "GIT_USER" }}'
+      branch: main
+
+actions:
+  opbeans-ruby:
+    kind: github/pullrequest
+    scmid: githubConfig
+    spec:
+      automerge: false
+      labels:
+        - dependencies
+
+sources:
+  elastic-apm-agent-ruby:
+    scmid: elastic-apm-agent-ruby
+    kind: gittag
+    spec:
+      versionfilter:
+        kind: semver
+    transformers:
+      - trimprefix: "v"
+
+targets:
+  dockerfile-label-schema:
+    name: Set org.label-schema.version in Dockerfile
+    sourceid: elastic-apm-agent-ruby
+    scmid: githubConfig
+    kind: file
+    spec:
+      file: Dockerfile
+      matchpattern: >-
+        org\.label-schema\.version="(\d+.\d+.\d+)"
+      replacepattern: >-
+        org.label-schema.version="{{ source `elastic-apm-agent-ruby` }}"
+  dockerfile-opencontainers:
+    name: Set org.opencontainers.image.version in Dockerfile
+    sourceid: elastic-apm-agent-ruby
+    scmid: githubConfig
+    kind: file
+    spec:
+      file: Dockerfile
+      matchpattern: >-
+        org\.opencontainers\.image\.version="(\d+.\d+.\d+)"
+      replacepattern: >-
+        org.opencontainers.image.version="{{ source `elastic-apm-agent-ruby` }}"
+  gem:
+    name: Bump elastic-apm
+    sourceid: elastic-apm-agent-ruby
+    scmid: githubConfig
+    kind: shell
+    spec:
+      command: ./.ci/bump-gem.sh
```

Comment:
Question:

I wonder whether this step could be replaced by:
- bumping the version in the Gemfile directly
- then run a new target in charge of:
  -  bundle update elastic-apm"
- configure the  `.github/workjflows/bump-opbeans-ruby.yml` to run within a ruby context https://github.com/ruby/setup-ruby

Reply:
I actually had something similar. but I wasn't 100% sure about it.

I was not able to find anything in the documentation, but do you know if I can mark a target dependent on another target in updatecli?

Reply:
I think targets in the updatecli run sequentially

`dependson` is available for targets -> https://www.updatecli.io/docs/core/target/

Reply:
Thank you.. I knew I saw it somewhere, but couldn't find it anymore.

https://github.com/elastic/opbeans-ruby/pull/171/commits/cdfa3183077144cc685873e6ab3ba7fe1f8cd3c4

Reply:
https://github.com/elastic/opbeans-ruby/pull/171/files/bec01713d36fa013fd1174dff276b4de5d336e8b..af3f812222f4add1dda53fba11c37032e704dc34